app/routes/git_tokens.py

```
@@ -106,3 +170,17 @@ async def get_token_by_label(label:str) ->List[Dict[str, Any]]:
             detail="Service temporarily unavailable. Please try again later."
         ) from e
 
+
+
+@router.post("/", response_model=Dict[str, Any])
+async def add_token(request: Request, payload: AddGitlabSchema = Body(...)):
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)